### PR TITLE
feat: update pies logic

### PR DESCRIPTION
**Pull request by d0gge**
Changed ids of pies.

### CODE REVIEW

File: skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts [hunk 1]
```diff
@@ -161,9 +161,9 @@ export class SweetPieGameModeListener implements GameModeListener {
           const percentages = this.controller.getPercentages(casterActorId);
           this.controller.setPercentages(
             casterActorId, {
-              health: percentages.health! + .5,
-              magicka: percentages.magicka! + .5,
-              stamina: percentages.stamina! + .5
+            health: percentages.health! + .5,
+            magicka: percentages.magicka! + .5,
+            stamina: percentages.stamina! + .5
```
Comment (nic11) [inline skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts:166]: ?

File: skymp5-server/cpp/server_guest_lib/ActionListener.cpp [hunk 2]
```diff
@@ -648,6 +649,14 @@ void ActionListener::OnHit(const RawMessageData& rawMsgData_,
     return;
   }
 
+  std::unordered_set<uint32_t> greenZones = { 0x0760AADA };
+  uint32_t aggressorCell =
+    aggressor->GetCellOrWorld().ToFormId(aggressor->GetParent()->espmFiles);
+  auto it = greenZones.find(aggressorCell);
+  if (it != greenZones.end()) {
+    return;
+  }
```
Comment (nic11) [inline skymp5-server/cpp/server_guest_lib/ActionListener.cpp:658]: Does it work or will it be replaced with an alternative implementation?

File: skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts (hunk 1 above)
Comment (d0gge) [reply skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts:166]: Formatter formatted this

File: skymp5-server/cpp/server_guest_lib/ActionListener.cpp (hunk 2 above)
Comment (d0gge) [reply skymp5-server/cpp/server_guest_lib/ActionListener.cpp:658]: Doesn't, I'll remove it asap

**Review by nic11 — APPROVED** [review]

**Pull request merged**